src/renderer/containers/Onboard/SigningKey/Import/FileUploadImport.tsx

```diff
@@ -53,7 +52,7 @@ export const FileUploadImport: FC<IOwnProps> = ({history}) => {
         if (isSwitching) {
             history.replace(Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.SIGNING_IMPORT_SLASHING_FILE));
         } else {
-            history.replace(Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.PASSWORD));
+            history.replace(Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.CONFIGURE));
```

Comment:
Didn't notice earlier. This will route user back to import screen if he clicks back?

Reply:
worked kinda weird but fix it to be more intuitive